src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -1550,3 +1550,30 @@ function Test-ApplicationGatewayCRUDSubItems2
 		Clean-ResourceGroup $rgname
 	}
 }
+
+function Test-AvailableServerVariableAndHeader
+{
+	#Get server variables, request headers and response headers
+	$result = Get-AzApplicationGatewayAvailableServerVariableAndHeader
+
+	Assert-NotNull $result
+	Assert-True { $result.Count -gt 0 }
```

Comment:
You should assert that all three dictionary entries are not empty
```powershell
Assert-True { $result.AvailableServerVariable.Count -gt 0 }
# and so on
```
Same goes for the following tests

Reply:
Done.